packages/webdriverio/src/utils/Timer.ts

```diff
@@ -2,17 +2,21 @@ import { hasWdioSyncSupport, runFnInFiberContext } from '@wdio/utils'
 
 const TIMEOUT_ERROR = 'timeout'
 
-/**
- * Promise-based Timer. Execute fn every tick.
- * When fn is resolved — timer will stop
- * @param {Number} delay - delay between ticks
- * @param {Number} timeout - after that time timer will stop
- * @param {Function} fn - function that returns promise. will execute every tick
- * @param {Boolean} leading - should be function invoked on start
- * @return {promise} Promise-based Timer.
- */
 class Timer {
-    constructor (delay, timeout, fn, leading) {
+    private _delay: number;
+    private _timeout: number;
+    private _fn: () => Promise<boolean> | boolean;
+    private _leading: boolean;
+    private _conditionExecutedCnt: number;
+    private _start?: number;
+    private _ticks?: number;
+    private _timeoutId?: NodeJS.Timeout | null;
+    private _mainTimeoutId?: NodeJS.Timeout | null;
+    lastError?: Error | null;
```

Comment:
Do we access the variable somewhere else? If not we can make it private too, no?
```suggestion
    private lastError?: Error;
```

Reply:
Apparently we don't access anywhere so I have the same question. I didn't make it `private` at first as _it seemed to me_ that the private variables were declared starting with an __underscore_

But making it private also doesn't seem to cause any problem!

Reply:
Oops, ok it seems it can't be private, broke some other tests.

Reply:
@Fabianopb to access the property in the unit tests just do `timer['_lastError']`